src/app_service/common/wallet_constants.nim

```diff
@@ -9,3 +9,5 @@ const
 
   TX_HASH_LEN* = 32 * 2
   TX_HASH_LEN_WITH_PREFIX* = TX_HASH_LEN + 2
+
+  PARASWAP_V5_CONTRACT_ADDRESS* = "0x1b40183efb4dd766f11bda7a7c3ad8982e998421" # Same address for all supported chains
```

Comment:
I guess this only till we have a list from backed. lets add a TODO with link to https://github.com/status-im/status-desktop/issues/15329 ?